plugins/woocommerce/tests/php/src/Utilities/StringUtilTest.php

```diff
@@ -74,4 +74,40 @@ public function test_plugin_name_from_plugin_file() {
 		$expected  = 'foobar/fizzbuzz.php';
 		$this->assertEquals( $expected, $result );
 	}
+
+	/**
+	 * @testDox 'is_null_or_empty' should return true only if the value is null or an empty string.
+	 *
+	 * @testWith [null, true]
+	 *           ["", true]
+	 *           ["  ", false]
+	 *           ["0", false]
+	 *           ["foo", false]
+	 *           ["  foo  ", false]
```

Comment:
Cool, I didn't know you could put the test data right in the doc block!

Reply:
[You are not the first one that learns this by reviewing my PRs](https://github.com/woocommerce/woocommerce/pull/36891#discussion_r1113491860). Apparently I'm the only one around here that reads manuals. 😄